apps/AppWithWearable/lib/env/env.dart

```diff
@@ -1,34 +1,35 @@
-import 'package:envied/envied.dart';
+import 'package:friend_private/env/prod_env.dart';
 
-part 'env.g.dart';
+import '../flavors.dart';
+import 'dev_env.dart';
 
-@Envied(allowOptionalFields: true)
 abstract class Env {
-  @EnviedField(varName: 'SENTRY_DSN_KEY', obfuscate: true)
-  static String? sentryDSNKey = _Env.sentryDSNKey;
-
-  @EnviedField(varName: 'OPENAI_API_KEY', obfuscate: true)
-  static String? openAIAPIKey = _Env.openAIAPIKey;
-
-  @EnviedField(varName: 'DEEPGRAM_API_KEY', obfuscate: true)
-  static String? deepgramApiKey = _Env.deepgramApiKey;
-
-  @EnviedField(varName: 'INSTABUG_API_KEY', obfuscate: true)
-  static String? instabugApiKey = _Env.instabugApiKey;
-
-  // Pinecone
-  @EnviedField(varName: 'PINECONE_API_KEY', obfuscate: true)
-  static String pineconeApiKey = _Env.pineconeApiKey;
-
-  @EnviedField(varName: 'PINECONE_INDEX_URL', obfuscate: true)
-  static String pineconeIndexUrl = _Env.pineconeIndexUrl;
-
-  @EnviedField(varName: 'PINECONE_INDEX_NAMESPACE', obfuscate: true)
-  static String pineconeIndexNamespace = _Env.pineconeIndexNamespace;
-
-  @EnviedField(varName: 'MIXPANEL_PROJECT_TOKEN', obfuscate: true)
-  static String? mixpanelProjectToken = _Env.mixpanelProjectToken;
+  static late final EnvFields _instance;
+
+  static void init() {
+    _instance = F.appFlavor == Flavor.dev ? DevEnv() : ProdEnv();
```

Comment:
Flavor can be passed here as a param from `init()` function throughout `main()`. Then we don't need checks here.

Reply:
Additionally, for more simplicity, `DevEnv()` or `ProdEnv()` can come from respective main file.